pkg/sensors/platform/source/redfish.go

```diff
@@ -270,7 +270,7 @@ func (rf *RedFishClient) IsSystemCollectionSupported() bool {
 			}
 		}
 	}()
-	return rf.systems != nil && len(rf.systems) > 0
+	return len(rf.systems) > 0
```

Comment:
👀 is this related?

Reply:
no I can pull it out into a separate PR... golangci-lint doesn't like it